README.md

```diff
@@ -54,3 +54,5 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Running Vagrant
+Need to run vagrant up
```

Comment:
This could be fleshed out a bit... Something like:
You can also launch the app inside a VM. Requirements: Vagrant and a hypervisor such as VirtualBox. Start it up by running the command `vagrant up` in a terminal at the root of the project. You can then access it by visiting http://localhost:5000/ in a browser.